doc/api/module.md

```diff
@@ -269,65 +164,224 @@ changes:
                  `globalPreload`; added `load` hook and `getGlobalPreload` hook.
 -->
 
-> Stability: 1 - Experimental
-
-> This API is currently being redesigned and will still change.
+> Stability: 1.1 - Active development
 
 <!-- type=misc -->
 
-To customize the default module resolution, loader hooks can optionally be
-provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
+<i id="enabling_module_customization_hooks"></i>
+
+### Enabling
+
+Module resolution and loading can be customized by registering a file which
+exports a set of hooks. This can be done using the [`register`][] method
+imported from `node:module`, which you can run before your application code by
+using the `--import` flag:
+
+```bash
+node --import ./register-hooks.js ./my-app.js
+```
+
+```mjs
+// register-hooks.js
+import { register } from 'node:module';
+
+register('./hooks.mjs', import.meta.url);
+```
+
+```cjs
+// register-hooks.js
+const { register } = require('node:module');
+const { pathToFileURL } = require('node:url');
+
+register('./hooks.mjs', pathToFileURL(__filename));
+```
+
+The file passed to `--import` can also be an export from a dependency:
+
+```bash
+node --import some-package/register ./my-app.js
+```
+
+Where `some-package` has an `"exports"` field defining the `/register` export to
+map to a file that calls `register()`, like this `register-hooks.js` example.
```

Comment:
I think we should either link to the `"exports"` documentation, or not mention it at all considering folks already know – and realistically, they will just copy the command from `some-package` README, so they don't have to care about exports map or this kind of stuff.

Reply:
I'm fine with not mentioning it, but will people be confused?